generator/template/assets/info-filled-icon.tsx

```diff
@@ -0,0 +1,14 @@
+import { SVGAttributes } from 'react'
+
+export const InfoFilledIcon = (props: SVGAttributes<SVGElement>) => (
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this related to all icons
```suggestion
<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```